### PR TITLE
Add trace docs to main docs index

### PR DESCRIPTION
**Pull request by liyanhui1228**
Fix the trace docs.

### CODE REVIEW

File: docs/trace/apis.rst [hunk 1]
```diff
@@ -5,7 +5,7 @@ APIs
 ----
 
 .. autosummary::
-   :toctree:
+.. :toctree::
```
Comment (dhermes) [inline docs/trace/apis.rst]: The `toctree` seems like it should be away from the `autosummary`?

File: docs/trace/starting.rst [hunk 2]
```diff
@@ -74,5 +74,20 @@ At this point you are all set to continue.
...
+  client.patch_traces(traces=traces)
+
+  # Get trace
+  client.get_trace(trace_id='your_trace_id')
+
+  # List traces
+  traces = client.list_traces()
+
+  for trace in traces:
+      print trace
```
Comment (dhermes) [inline docs/trace/starting.rst]: Please use a Python 3 friendly `print(trace)`

File: docs/trace/apis.rst (hunk 1 above)
Comment (liyanhui1228) [inline docs/trace/apis.rst]: Does it mean adding a blank line between them? Or move it somewhere?

File: docs/trace/starting.rst (hunk 2 above)
Comment (liyanhui1228) [inline docs/trace/starting.rst]: Done.

File: docs/trace/apis.rst (hunk 1 above)
Comment (dhermes) [inline docs/trace/apis.rst]: I'm not sure. Did you write these docs? What was the intent?

Comment (liyanhui1228) [inline docs/trace/apis.rst]: This part is auto generated. Not sure the intent.

Comment (dhermes) [inline docs/trace/apis.rst]: @lukesneeringer What did the autogen tool intend here?

**Pull request merged**